ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -53,14 +53,14 @@ jobs:
         timeout: 1800
         topology: *build
 
-  fedora-30/temp_commit:
+  fedora-30/test_commands:
     requires: [fedora-30/build]
     priority: 50
     job:
       class: RunPytest
       args:
         build_url: '{fedora-30/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_commands.py
         template: *ci-master-f30
         timeout: 3600
-        topology: *master_1repl_1client
+        topology: *master_1client
```

Comment:
This topology does not exist, you should use master_1repl instead.
Full explanation: the topology specified in the yaml file with `topology: *<topology_name>` must be defined in the same yaml file in the `topologies:` section. The char & is used to denote an anchor and the char * is used to denote an alias, ie avoid typing twice the same content.

Reply:
topology has been modified to  topology: *master_1repl